tickets/fixtures/moderation_queues.json

```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "pk": 1,
+    "model": "tickets.queue",
+    "fields": {
+      "notify_by_email": false,
+      "name": "sound moderation",
+      "groups": [
+        3
```

Comment:
can this use the natural key to get the group instead of a pk?

Reply:
Done in f5e643998d1350c2a37c8d9a7c4ab2fe91397051